Scripts/jazzy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Generates Jazzy documentation: https://github.com/realm/jazzy
+
+jazzy \
+  --objc \
+  --author Ably \
+  --module-version 1.2.10 \
```

Comment:
We need to keep the `--module-version` argument in sync with the current library version. We can do this either by:

- adding this file to `Scripts/set-version.sh`, or
- using `Scripts/get-version.sh` inside this script to populate the `--module-version` argument.